tailor_distro/manage/base.py

```diff
@@ -31,15 +36,101 @@ def insert_auth_token(url, token):
     return urlunsplit(parts)
 
 
+RAW_GH_PATTERN = re.compile('https://raw.githubusercontent.com/([^/]+/[^/]+)/([^/]+)/(.*)')
+
+
+def load_private_raw_file(url):
+    m = RAW_GH_PATTERN.match(url)
+    if not m:
+        raise RuntimeError('Cannot parse raw github url: {}'.format(url))
+    repo, branch, path = m.groups()
+    gh = get_github_client()
+    repo = gh.get_repo(repo)
+    git_file = repo.get_contents(path, branch)
+    yaml_str = git_file.decoded_content.decode()
+    return yaml.safe_load(yaml_str)
+
+
+def write_private_raw_file(url, new_contents, message='[no message given]'):
```

Comment:
instead of `private`, can we call these methods `github`, since that's the interface they implement?